style.css

```diff
@@ -113,3 +113,146 @@ header {
   list-style-type: none;
   padding-bottom: 5rem;
 }
+
+.projects {
+  margin: 20% 6% 10% 6%;
+}
```

Comment:
- [ ] Please I suggest adding CSS `display: grid` to the `projects` class per the project requirements. 😄